Rakefile

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+require 'jekyll'
+require 'html-proofer'
+
+# Extend string to allow for bold text.
+class String
+  def bold
+    "\033[1m#{self}\033[0m"
+  end
+end
+
+# Rake Jekyll tasks
+task :build do
+  puts 'Building site...'.bold
+  Jekyll::Commands::Build.process(profile: true)
+end
+
+task :clean do
+  puts 'Cleaning up _site...'.bold
+  Jekyll::Commands::Clean.process({})
+end
+
+# Test generated output has valid HTML and links.
+task :test do
+  Jekyll::Commands::Build.process(profile: true)

```

Comment:
Thinking twice about this: Isn't it possible to declare that one Rake task depends on another, somehow? `:test` depends on `:build` would be the proper way to do this, I think.

Reply:
Can do!